tests/integration/vlan/41-vlan-bridge-native.yml

```diff
@@ -12,15 +12,22 @@ message: |
   Please note it might take a while for the lab to work due to
   STP learning phase
 
+plugin: [ 41-s2-green ]
+
 groups:
   _auto_create: True
   hosts:
     members: [ h1, h2, h3, h4, h5, h6 ]
     device: linux
     provider: clab
   switches:
-    members: [ s1, s2 ]
+    members: [ s1 ]
+    module: [ vlan ]
+  probes:
+    members: [ s2 ]
     module: [ vlan ]
+    device: eos
```

Comment:
Could ```frr``` be used too?

EOS has a hardened, production grade implementation, FRR is open source and more easily obtained.
Could add a comment explaining why ```eos``` is preferred here

Reply:
Let's just say I trust EOS more than what we're hacking together for FRR ;)